jonathan/settings.py

```diff
@@ -0,0 +1,129 @@
+"""
+Django settings for jonathan project.
+
+Generated by 'django-admin startproject' using Django 2.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+from decouple import config
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'ahe(urmoht$=0x^_qq#l3#b74yu%$8w)ck%ulbf1(3a$*5+hq@'
```

Comment:
you're defining 'SECRET_KEY' twice. and you've exposed the key. It should only be in your `.env` file